stable/nginx-ingress/templates/controller-deployment.yaml

```diff
@@ -22,6 +25,9 @@ spec:
         component: "{{ .Values.controller.name }}"
         release: {{ .Release.Name }}
     spec:
+{{- if .Values.controller.rbac.enabled }}
+      serviceAccountName: {{ template "fullname" . }}
+{{- end }}
```

Comment:
No, I didn't experience this. I would presume the issue is elsewhere in your case, as this conditional logic is very simple.